lightcurve/views.py

```diff
@@ -113,6 +114,16 @@ def add_image_toggle(request, lightcurve_id):
         'success': True,
     })
 
+def save_lightcurve_changes(request, lightcurve_id):
+    filename = request.POST.get('input_name')
```

Comment:
Let's rename this var to something other than `filename` - maybe just `name`?

Reply:
Done.